package-lock.json

```diff
@@ -7301,6 +7302,7 @@
       "resolved": "https://registry.npmjs.org/are-we-there-yet/-/are-we-there-yet-1.1.7.tgz",
       "integrity": "sha512-nxwy40TuMiUGqMyRHgCSWZ9FM4VAoRP4xUYSTv5ImRog+h9yISPbVH7H8fASCIzYn9wlEv4zvFL7uKDMCFQm3g==",
       "dev": true,
+      "optional": true,
```

Comment:
Thanks for contributing to this project, sorry for the delay, could you please revert the changes in this file? since you just changed the readme, I don't think the package-lock.json should be part of this PR.

Thanks!